README.md

```diff
@@ -48,3 +49,61 @@ gem install mongoid_includes
 ## License
 
 The gem is available as open source under the terms of the [MIT License](https://opensource.org/licenses/MIT).
+
+## Running tests
+
+To run the full test suite locally:
+
+```sh
+bundle install
+bundle exec rspec
+# or use the bundled binary
+bin/rspec
```

Comment:
You can run `rake` which does it too. I think it's because the default task is to run the rspec tests.